cypress/integration/system_console/demoted_user_spec.js

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+/*eslint max-nested-callbacks: ["error", 3]*/
+/*eslint-disable func-names*/
+
+describe('Demoted user', () => {
+    beforeEach(() => {
+        cy.fixture('users').as('usersJSON');
+
+        cy.get('@usersJSON').
+            its('sysadmin').
+            as('sysadmin');
+
+        cy.get('@usersJSON').
+            its('user-1').
+            as('user');
```

Comment:
While it looks like a Cypress way of getting data from fixture, I find it not the best use in this case and it reads more complicated than just getting it directly, like:

```javascript
import users from '../../fixtures/users.json';
...
const user = users['user-1'];
const sysadmin = users['sysadmin'];
```

Also, it gets a promise - ``cy.get('@user').then(...)`` - to initiate rather than doing it directly as ``cy.login(user.username)`` 

Would love to hear your thoughts on it.

Reply:
For sure it's feels like the "cypress-ey" way to do it and that was the main motivator for this approach, but getting a second opinion on this leads me to agree with you. The aliasing in this scenario feels foreign and we really don't gain anything from it, while other use cases (elements, non-json/js fixtures, etc...) I feel like it's much more usable. I'll fix it up.